workflow-templates/im-test-k6-operator-approval.yml

```
@@ -80,10 +80,10 @@ permissions:
 env:
   TIMEZONE: 'america/denver' # TODO: Verify timezone
   WORKING_DIRECTORY: './tests/k6' # TODO: Update k6 test folder path
-  GRAFANA_DASHBOARD_URI: 'https://grafana.mktp.io/d/L8Ilp7C7k/k6-load-testing-results-product-tag-filter?orgId=1&from=now-1h&to=now&var-ProductTag=<your-product-tag>' # TODO: Verify link to your dashboard
+  GRAFANA_DASHBOARD_URI: 'https://grafana.mktp.io/d/SizO_pC4k/k6-load-testing-results-with-env?orgId=1?orgId=1&from=now-1h&to=now&var-ProductTag=<your-product-tag>&var-env=${{inputs.env}}' # TODO: Update product tag
```

Removes the extra `?orgId=1`
```suggestion
  GRAFANA_DASHBOARD_URI: 'https://grafana.mktp.io/d/SizO_pC4k/k6-load-testing-results-with-env?orgId=1&from=now-1h&to=now&var-ProductTag=<your-product-tag>&var-env=${{ inputs.env }}' # TODO: Update product tag
```